### PR TITLE
Minor followup updates

### PR DESCRIPTION
**Pull request by gfcas**
@koecher These are some minor updates on the platform files I already did some weeks ago (now rebased on #218). In general its only indention (via shfmt) and spelling. I just realize that you deleted ubuntu19 instead of shifting into deprecated. Do you want to recover it, or should we remove all 'end of life' distributions? As ubuntu16 should be at the 'end of life' I moved it into deprecated.

### CODE REVIEW

**Comment by koecher** [comment]
yes, I deleted the `ubuntu19.platform` file instead of moving. Anyhow, platforms in `deprecated` aren't automatically detected.

**Comment by koecher** [comment]
I would prefer to firstly decide for a new format style and set a script file to the repo. Then one clean commit after running this script. The platform files should be reviewed firstly which are really needed anymore.

**Comment by gfcas** [comment]
> 
> 
> yes, I deleted the `ubuntu19.platform` file instead of moving. Anyhow, platforms in `deprecated` aren't automatically detected.

Sorry, but this seems to me a bit inconsistent... Maybe keeping only the LTS version as deprecated? (Removing all deprecated platforms is maybe too hard?)

**Comment by gfcas** [comment]
> 
> 
> I would prefer to firstly decide for a new format style and set a script file to the repo. Then one clean commit after running this script. The platform files should be reviewed firstly which are really needed anymore.

I totally agree, in running a indention script once on all files. Here the IDE accidently runs the auto-indention accidently, however, basically only some trailing withespaces were removed. With this PR I wanted to fix some typos.

**Comment by koecher** [comment]
> > yes, I deleted the `ubuntu19.platform` file instead of moving. Anyhow, platforms in `deprecated` aren't automatically detected.
> 
> Sorry, but this seems to me a bit inconsistent... Maybe keeping only the LTS version as deprecated? (Removing all deprecated platforms is maybe too hard?)

indeed it is inconsistent. The deletion was a mistake and can be undone. I see two options:
- 1st, make the detection from systems of the deprecated folder possible and warn on the deprecation. 
- 2nd, remove the deprecated folder at all.

For Ubuntu, we may go into the way I've started for Fedora, except for special versions use a `ubuntu.platform`.

**Comment by gfcas** [comment]
Essentially, I think it makes sense to follow the Ubuntu release policy, since no one should be using a security-critical system anymore. Therefore, I would delete the deprecated platform files in the master branch. If someone really still wants to install an old dealii version on an outdated system, then those files are still in the old branches. So for this PR I deleted all 'end of life' ubuntu files.

Having one ubuntu.platform file would be of course nice. However, I observed that the required packages list is slightly different. In particular, Ubuntu 18.04 requires additionally
```
#   splint tcl tcl-dev environment-modules qt4-dev-tools
```
Are those really necessary? In case they can be omitted, a unified ubuntu.platform file is provided in this PR.

Furthermore, according to @tjhei's comment in #218 the compiler issue is more general. Thus let us rewrite the information a bit more general.

Finally, using the candi-cmake packages seems no longer necessary (the github actions runners with cmake 3.20.5 are happy), so we could remove this comment from the platform file.

**Review by tjhei — APPROVED** [review]

**Comment by koecher** [comment]
> Essentially, I think it makes sense to follow the Ubuntu release policy, since no one should be using a security-critical system anymore. Therefore, I would delete the deprecated platform files in the master branch. If someone really still wants to install an old dealii version on an outdated system, then those files are still in the old branches. So for this PR I deleted all 'end of life' ubuntu files.

I think the deletion of the platform files would be nicer to have them in a one-commit PR.

> Having one ubuntu.platform file would be of course nice. However, I observed that the required packages list is slightly different. In particular, Ubuntu 18.04 requires additionally
> 
> ```
> 
> #   splint tcl tcl-dev environment-modules qt4-dev-tools
> 
> ```
> 
> Are those really necessary? In case they can be omitted, a unified ubuntu.platform file is provided in this PR.
> 

Not mandatory but useful. `splint` is for `p4est`, the next three are `module`(files) and `qt4` isn't needed anymore, this was for the deprecated gui-editor of prm files.

File: deal.II-toolchain/platforms/supported/ubuntu.platform [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# If the default compiler is older than the hardware generation, a newer
+# compiler version is necessary to use native optimizations.
+# For example, to update the GNU Compiler Collection from the default v9 to
+# v10, install the new compilers
+#
+# > sudo apt-get install gcc-10 g++-10 gfortran-10
+#
+# and update the symbolic links to gcc, g++ and gfortran by
+#
+# > update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-9 10
```
[the remark this answers is not in the document]
Comment (gfcas) [reply deal.II-toolchain/platforms/supported/ubuntu.platform:23]: I understand and undo it

File: candi.sh [hunk 2]
```diff
@@ -600,8 +600,8 @@ guess_platform() {
         echo cygwin
 
     elif [ -x /usr/bin/sw_vers ]; then
-        local MACOSVER=$(sw_vers -productVersion)
-        case ${MACOSVER} in
+        local MACOS_VERSION=$(sw_vers -productVersion)
+        case ${MACOS_VERSION} in
```
[the remark this answers is not in the document]
Comment (gfcas) [reply candi.sh:604]: OK

**Comment by gfcas** [comment]
> > Essentially, I think it makes sense to follow the Ubuntu release policy, since no one should be using a security-critical system anymore. Therefore, I would delete the deprecated platform files in the master branch. If someone really still wants to install an old dealii version on an outdated system, then those files are still in the old branches. So for this PR I deleted all 'end of life' ubuntu files.
> 
> I think the deletion of the platform files would be nicer to have them in a one-commit PR.

See #235

**Comment by gfcas** [comment]
> > ```
> > #   splint tcl tcl-dev environment-modules qt4-dev-tools
> > ```
> > Are those really necessary? In case they can be omitted, a unified ubuntu.platform file is provided in this PR.
> 
> Not mandatory but useful. `splint` is for `p4est`, the next three are `module`(files) and `qt4` isn't needed anymore, this was for the deprecated gui-editor of prm files.

If not mandatory, lets skip them in order to keep the unified platform file for Ubuntu as simple as possible?!

**Comment by gfcas** [comment]
@koecher I removed every deeper change from this PR and only kept the minor ones. Maybe we need to first merge #235, #236, #237 and then rebase this PR.

**Review by koecher — APPROVED** [review]
Thank you very much for this tedious work.

**Comment by koecher** [comment]
@gfcas Ah snap, can you please resolve the merge issue with `centos7.platform` and `debian9.platform`
Then this is ready to merge

**Comment by gfcas** [comment]
> 
> 
> @gfcas Ah snap, can you please resolve the merge issue with `centos7.platform` and `debian9.platform`
> Then this is ready to merge

Just rebasing ;)

**Review by koecher — APPROVED** [review]

**Comment by gfcas** [comment]
@koecher rebased.

**Comment by gfcas** [comment]
Thanks!

**Pull request merged**